SimPEG/PF/GravityDriver.py

```diff
@@ -275,15 +277,15 @@ def readGravityObservations(self, obs_file):
 
         # First line has the number of rows
         line = fid.readline()
-        ndat = np.array(line.split(), dtype=int)
+        ndat = int(line.strip())
```

Comment:
@fourndo : I updated how you are casting this to an `int` so that `ndat` is a true integer, not an `np.int64`